flexget/plugins/daemon/web_server.py

```diff
@@ -100,11 +102,14 @@ def register_web_server(manager):
     # Register WebUI
     if web_server_config.get('web_ui'):
         if web_server_config.get('run_v2'):
-            logger.info('Registering WebUI v2')
-            register_web_ui_v2(web_server_config)
+            logger.warning('Run V2 is deprecated and can be removed from the config')
```

Comment:
We have a way to set a field as deprecated I think, not sure it'll work on a root level schema though... Better ask @gazpachoking